package.json

```diff
@@ -59,6 +59,7 @@
     "arg": "^5.0.2",
     "clean-css": "^5.3.3",
     "magic-string": "^0.30.8",
+    "ora": "^5.4.1",
```

Comment:
ora v6 is native ESM, where `@swc-node/register` breaks.
Since we're using a simple indicator, we can stick to v5.4.1 IMO.